cli/common.go

```diff
@@ -18,3 +18,53 @@ type CommonFlags struct {
 	TLSOptions *tlsconfig.Options
 	TrustKey   string
 }
+
+// Command is the struct contains command name and description
+type Command struct {
+	Name        string
+	Description string
+}
+
+// DockerCommands stores all the docker command
+var DockerCommands = map[string]Command{
+	"attach":  {"attach", "Attach to a running container"},
```

Comment:
@duglin  @vdemeester  sorry for late response
I want remove the duplication command name, I'll update later